next.js build error on new workspace: can't find `node-sass`

_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

## Expected Behavior

It should build/serve succesfully.

## Current Behavior

There's an error:

```
./pages/index.scss
Error: Cannot find module 'node-sass'
Require stack:
- ......./test2/node_modules/sass-loader/lib/loader.js
- ......./test2/node_modules/loader-runner/lib/loadLoader.js
- ......./test2/node_modules/loader-runner/lib/LoaderRunner.js
- ......./test2/node_modules/webpack/lib/NormalModule.js
- ......./test2/node_modules/webpack/lib/NormalModuleFactory.js
- ......./test2/node_modules/webpack/lib/Compiler.js
- ......./test2/node_modules/webpack/lib/webpack.js
- ......./test2/node_modules/next/dist/build/compiler.js
- ......./test2/node_modules/next/dist/build/index.js
- ......./test2/node_modules/@nrwl/next/src/builders/build/build.impl.js
- ......./test2/node_modules/@angular-devkit/architect/node/node-modules-architect-host.js
- ......./test2/node_modules/@angular-devkit/architect/node/index.js
- ......./test2/node_modules/@nrwl/tao/src/commands/run.js
- ......./test2/node_modules/@nrwl/tao/index.js
- ......./test2/node_modules/@nrwl/cli/lib/init-local.js
- ......./test2/node_modules/@nrwl/cli/bin/nx.js
- /Users/raresmatei/.npm-global/lib/node_modules/@nrwl/cli/lib/init-global.js
- /Users/raresmatei/.npm-global/lib/node_modules/@nrwl/cli/bin/nx.js
```

### Steps to Reproduce

```bash
npx create-nx-workspace@latest repro-workspace ---> select next.js app
cd test1
nx build
```

`node-sass` doesn't exist under `node_modules`
If I do `yarn add node-sass` it works again.

### Context

Please provide any relevant information about your setup:

  @nrwl/angular : Not Found
  @nrwl/cli : 9.0.4
  @nrwl/cypress : 9.0.4
  @nrwl/eslint-plugin-nx : 9.0.4
  @nrwl/express : Not Found
  @nrwl/jest : 9.0.4
  @nrwl/linter : 9.0.4
  @nrwl/nest : Not Found
  @nrwl/next : 9.0.4
  @nrwl/node : Not Found
  @nrwl/react : 9.0.4
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.0.4
  @nrwl/web : 9.0.4
  @nrwl/workspace : 9.0.4
  typescript : 3.7.5

--- a/packages/next/src/schematics/application/lib/add-style-dependencies.ts
+++ b/packages/next/src/schematics/application/lib/add-style-dependencies.ts
@@ -6,7 +6,8 @@ import {
   zeitNextCss,
   zeitNextLess,
   zeitNextSass,
-  zeitNextStylus
+  zeitNextStylus,
+  nodeSass
 } from '../../../utils/versions';
 
 const NEXT_SPECIFIC_STYLE_DEPENDENCIES = {
@@ -15,7 +16,9 @@ const NEXT_SPECIFIC_STYLE_DEPENDENCIES = {
     dependencies: {
       '@zeit/next-css': zeitNextCss
     },
-    devDependencies: {}
+    devDependencies: {
+      'node-sass': nodeSass
+    }
   },
   scss: {
     dependencies: {

--- a/packages/next/src/utils/versions.ts
+++ b/packages/next/src/utils/versions.ts
@@ -3,5 +3,6 @@ export const nxVersion = '*';
 export const nextVersion = '9.3.3';
 export const zeitNextCss = '1.0.1';
 export const zeitNextSass = '1.0.1';
+export const nodeSass = '4.13.1';
 export const zeitNextLess = '1.0.1';
 export const zeitNextStylus = '1.0.1';

